elfpy/math/fixed_point.py

```diff
@@ -150,7 +153,7 @@ def __rmul__(self, other: int | FixedPoint) -> FixedPoint:
     def __truediv__(self, other):
         """Enables '/' syntax.
 
-        We mirror floordiv because most solidity contract equations use divdown
+        We use div_down to match the majority of Hyperdrive solidity contract equations
```

Comment:
We want to use the syntax sugar whenever possible. That being said, if a situation comes up where it really makes sense to call it `div_down` then I think it's ok to add it to `FixedPoint`. Notice we already have an example because we have an explicit `div_up`.